index.js

```diff
@@ -121,19 +121,13 @@ module.exports = {
             if (!node.value) return;
             const value = node.value.cooked;
             checkString(value, node);
+          },
+          JSONLiteral(node){
```

Comment:
`jsonc-eslint-parser` uses `JSONLiteral` for both key strings and value strings. So I believe it is more safe to add an additional condition to check that `JSONLiteral` is a value and not a JSON object key name. As strings also may be used in array I think it is easier just to exclude cases when literal is a key.

Reply:
Hey @constgen, 
Thanks for the feedback. Kind of a different API than I'm use to, and it does not seem to indicate if a value is a key or not. I tried doing some esquery magic with `JSONProperty > JSONLiteral`, but no luck. Do you have any ideas?
Best.
Nick